app/services/importer/workable_fields_formatter.rb

```diff
@@ -51,24 +51,18 @@ def attribute(raw_attribute)
         default_attribute(raw_attribute)
     end
 
-    def attribute_strict_match(key)
-      ATTRIBUTES_DICTIONARY[key]
-    end
-
-    # attribute is CamelCase variable or a Text Question
-    def default_attribute(key)
-      key.underscore.first(60).gsub(' ', '_').gsub('.', '_')
-    end
+    def attributes_dictionary = ATTRIBUTES_DICTIONARY
 
     ATTRIBUTES_DICTIONARY = {
```

Comment:
Can we discuss the difference between the ATTRIBUTES_DICTIONARY & INPUT_TYPES. I would like to understand how this differs across ATS systems